app/controllers/api/v1x3/progress_messages_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1x3
+    class ProgressMessagesController < ApplicationController
+      include Mixins::IndexMixin
+
+      def index
+        if params[:order_id]
```

Comment:
May we use `safe_params_for_list.require(:messagable_type)`?

Reply:
Yes, changed to used it.